lib/Runtime/Language/ModuleNamespace.cpp

```diff
@@ -338,28 +330,4 @@ namespace Js
         }
         return sortedExportedNames;
     }
-
-    Var ModuleNamespace::EntrySymbolIterator(RecyclableObject* function, CallInfo callInfo, ...)
```

Comment:
This alone doesn't affect the iteration order. We have a separate issue to address the iteration order: [https://github.com/Microsoft/ChakraCore/issues/3246](https://github.com/Microsoft/ChakraCore/issues/3246)

---
In reply to: [126488250](https://github.com/Microsoft/ChakraCore/pull/3307#discussion_r126488250) [](ancestors = 126488250)